portal/src/main/java/org/bonitasoft/console/client/user/process/action/ProcessInstantiationCallbackBehavior.java

```diff
@@ -25,32 +25,45 @@ protected void redirectToCaseMoredetails(final String caseId) {
         History.newItem("?_p=" + CaseMoreDetailsPage.TOKEN + "&id=" + caseId + "&_pf=" + ClientApplicationURL.getProfileId());
     }
 
+    protected void redirectToCurrentProfileHome() {
+        History.newItem("?_pf=" + ClientApplicationURL.getProfileId());
+    }
+
     public void onSuccess(final String responseContent) {
+        if (ViewController.hasOpenedPopup()) {
+            ViewController.closePopup();
+        }
+        final String caseId = getCaseId(responseContent);
+        if (caseId != null) {
+            showConfirmation(_("The case %caseId% has been started successfully.", new Arg(
+                    "caseId", caseId)));
+            //Wait a short delay before redirecting. This is useful in case the process completes right away (archiving may take a while)
+            final Timer timer = new Timer() {
+
+                @Override
+                public void run() {
+                    redirectToCaseMoredetails(caseId);
+                }
+            };
+            timer.schedule(DELAY_BEFORE_REDIRECTION);
+        } else {
+            GWT.log("caseId is not set");
+            showConfirmation(_("A case has been started successfully."));
```

Comment:
No, it goes in the onFailure method instead (because the status sent in the post message is an error)